tests/test_integration.py

```diff
@@ -158,6 +158,7 @@ def test_element_str_from_highavailability_with_ha1_and_ha2_children(self):
             b'<description>my ha conf description</description>',
             b'<configuration-synchronization><enabled>yes</enabled>',
             b'</configuration-synchronization><peer-ip>10.5.1.5</peer-ip>',
+            b'<peer-ip-backup>10.6.1.5</peer-ip-backup>',
```

Comment:
Yep, this is why `peer_ip_backup` needs to be at the end of the class definition.